webpack/assets/javascripts/bundle.js

```diff
@@ -46,7 +45,6 @@ window.tfm = Object.assign(window.tfm || {}, {
   hosts,
   httpProxies,
   toastNotifications,
-  numFields,
```

Comment:
I can imagine having in this file
```js
const numFieldsDeprecationOnly = {
  initAll: () => { window.tfm.tools.deprecate('initAll()', 'does nothing as of now, please stop calling it',    '3.2') };
};
```
And then here: `numFields: numFieldsDeprecationOnly,` Which would prevent plugins from failing hard on the function missing for two more version.